### PR TITLE
Update vault cert docs for http-01 challenge support

### PR DESCRIPTION
**Pull request by SpComb**
See https://github.com/kontena/kontena/pull/3212

### CODE REVIEW

File: using-kontena/vault.md [hunk 1]
```diff
@@ -69,24 +69,50 @@ The email is needed for Let's Encrypt to notify when certificates are about to e
...
+### Create domain authorization
 
-To be able to request certificates for a domain you must first prove that you are in charge of that domain. For this, Kontena certificate management supports both DNS and TLS-SNI verifications.
+In order to request a Let's Encrypt certificate for any domains, you must first prove that you are in control of each domain.
+Kontena's certificate management for Let's Encrypt supports different challenge types:
 
-**TLS-SNI based verification**
+* `dns-01` (since <1.0)
+* `tls-sni-01` (since 1.4)
+* `http-01` (since 1.4.x/1.5)
```
Comment (SpComb) [inline using-kontena/vault.md]: Pending decision on https://github.com/kontena/kontena/pull/3212 backport

File: using-kontena/vault.md [hunk 2]
```diff
@@ -69,24 +69,52 @@ The email is needed for Let's Encrypt to notify when certificates are about to e
...
+
+Kontena automatically provisions the acme-challenge token/key-authorizations provided by LE to the linked service in the form of `ACME_CHALLENGE_*` environment variable secrets.
+The linked service is typically a [Kontena Load Balancer](loadbalancer.md) service which responds to `GET /.well-known/acme-challenge` requests using the provisioned tokens.
+
+Later on when requesting the certificate, the Let's Encrypt verification servers will connect to the domain on TCP port 80, and make a HTTP request for the challenge token.
+The `--linked-service` must be configured to expose port 80.
+
+#### TLS-SNI based verification
+
+The `tls-sni-01` challenge can also be used for automated certificate renewals.
```
Comment (SpComb) [inline using-kontena/vault.md]: Should this have a warning that `tls-sni-01` challenges are currently not working?

Comment (jnummelin) [reply using-kontena/vault.md]: sure. maybe with a link to the status blog/forum on LE side?

Comment (SpComb) [reply using-kontena/vault.md]: Linked to https://github.com/kontena/kontena/issues/3209, which has links to the LE status/blog

File: using-kontena/vault.md [hunk 3]
```diff
@@ -69,24 +69,54 @@ The email is needed for Let's Encrypt to notify when certificates are about to e
...
+### Authorize the domain
 
-To be able to request certificates for a domain you must first prove that you are in charge of that domain. For this, Kontena certificate management supports both DNS and TLS-SNI verifications.
+In order to request a Let's Encrypt certificate for any domains, you must first prove that you are in control of each domain.
+Kontena's certificate management for Let's Encrypt supports different challenge types:
 
-**TLS-SNI based verification**
+* `dns-01` (since <1.0)
+* `tls-sni-01` (since 1.4, but [out of service](https://github.com/kontena/kontena/issues/3209) since 2018-01-10)
+* `http-01` (since 1.4.x/1.5)
```
Comment (SpComb) [inline using-kontena/vault.md]: Looking like this should be just `1.5`

**Comment by SpComb** [comment]
Bump for merging/updating for the 1.5 release.

**Comment by SpComb** [comment]
PTAL docs for 1.5.0 release

**Review by jakolehm — APPROVED** [review]

**Pull request merged**